backend/test/questionRoute.test.js

```diff
@@ -13,15 +13,18 @@ describe('Test Question Route', function () {
     test('getQuestion', async () => {
         const res = await request(app).get('/');
         expect(res.header['content-type']).toBe('application/json; charset=utf-8');
-        expect(res.statusCode).toBe(200);
+        expect(res.status).toBe(200);
```

Comment:
Interesting, I think statusCode should be the way to test the backend results per: https://stackoverflow.com/questions/46129307/difference-between-response-statuscode-and-response-status

Was statusCode failing for you or did you have some reasoning in mind for this?

Reply:
Oh sorry that makes sense, I just thought that since we're only calling res.status, we should test the status itself but I didn't know that it's corresponding to the server-side. You can disregard it